modules/editor/editor.js

```diff
@@ -1,8 +1,45 @@
+import InserterButton from './inserter/button';
+
+const h = wp.element.createElement;
+
+const EditorComponent = ( { content, state: { inserter }, toggleInserter } ) => {
```

Comment:
Something bothers me about components not being in their own standalone file. Maybe I'm overthinking it. Alternative might be `editor/index.js` (raw class) and `editor/editor.js` (component) but then the path becomes `editor/editor/editor.js` 😬